pkg/services/ngalert/state/manager_test.go

```diff
@@ -1868,7 +1868,7 @@ func TestProcessEvalResults(t *testing.T) {
 							Values:          make(map[string]*float64),
 						},
 					},
-					StartsAt:           evaluationTime.Add(20 * time.Second),
+					StartsAt:           evaluationTime.Add(30 * time.Second),
```

Comment:
This I think is a mistake in the test. The timestamp of the first evaluation error is at `evaluationTime.Add(30 * time.Second)`, not `evaluationTime.Add(20 * time.Second)`.